api/src/paths/dwc/spatial/search.ts

```diff
@@ -78,8 +90,6 @@ export function searchSpatialComponents(): RequestHandler {
 
       const response = await spatialService.findSpatialComponentsByCriteria(criteria);
 
-      await connection.commit();
-
```

Comment:
I think we still want this. At the very least just for consistency with other cases.

The connection.open starts a "BEGIN" transaction block, and the connection.commit does the closing "COMMIT".

I'm not entirely sure what it means to open a block, do only reads, and then not close it.  Maybe its fine? My worry would be if it left a dangling transaction or something that ate up resources until it eventually timed out.